### PR TITLE
(DOCSP-10331): Task Tracker Tutorial (Web)

### PR DESCRIPTION
**Pull request by nlarew**
**Tutorial Code PR: https://github.com/mongodb-university/realm-tutorial/pull/9**

## Jira

- [(DOCSP-10331): Task Tracker Tutorial (Web)](DOCSP-10331)

## Staged

- [Task Tracker (Web)](https://docs-mongodbcom-staging.corp.mongodb.com/realm/nlarew/web-tutorial/tutorial/web-graphql.html)

## To Do

- [x] ~Flesh out or remove Realm CLI instructions~ **Update: Removed**
- [x] Add content to the **What’s Next?** section (where should we point?)
- [x] Add a schema for Project & update other schemas for cross-tutorial sync
- [x] Update code snippets with updated data models
- [x] Split out backend tutorial
- [ ] Update screenshots that still refer to Stitch

### CODE REVIEW

File: source/tutorial/web-graphql.txt [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+
+.. admonition:: React Components
+   :class: note
+
+   React is a `popular
+   <https://insights.stackoverflow.com/survey/2020#technology-most-loved-dreaded-and-wanted-web-frameworks>`__
+   modern web application framework that uses a component model to maintain
+   application state and intelligently render pieces of the UI. If you're not
+   familiar with React or want to brush up on your knowledge, check out the
+   `official React website <https://reactjs.org>`_ which has excellent
```
Comment (cbush) [inline source/tutorial/web-graphql.txt:121]:
```suggestion
   `official React website <https://reactjs.org>`_, which has excellent
```

File: source/tutorial/web-graphql.txt [hunk 2]
```diff
@@ -0,0 +1,449 @@
...
+  - support multiple projects that each have their own set of tasks
+
+- Follow another tutorial to build a mobile app for the task tracker. We have
+  task tracker tutorials for the following platforms:
+  
+  - :doc:`iOS (Swift) </tutorial/ios-swift>`
+  - :doc:`Android (Kotlin) </tutorial/android-kotlin>`
+  - :doc:`React Native (JavaScript) </tutorial/react-native>`
+
+- Diver deeper into the docs to learn more about MongoDB Realm. You'll find
```
Comment (cbush) [inline source/tutorial/web-graphql.txt:430]:
```suggestion
- Dive deeper into the docs to learn more about MongoDB Realm. You'll find
```

File: source/tutorial/web-graphql.txt [hunk 3]
```diff
@@ -0,0 +1,449 @@
...
+.. cssclass:: bordered-figure
+.. figure:: /images/task-tracker-web-graphql-logs.png
+   :width: 750px
+
+What's Next?
+------------
+
+You just built a functional task tracker web application built with MongoDB
+Realm. Great job!
+
```
Comment (cbush) [inline source/tutorial/web-graphql.txt:413]: I've been writing this at the end of the other tutorials. Who knows?

>How did it go? Please leave feedback using the feedback widget on the bottom right of the page.

Comment (nlarew) [reply source/tutorial/web-graphql.txt:413]: Sure why not

**Pull request merged**